Server/src/main/java/org/openas2/lib/helper/ICryptoHelper.java

```diff
@@ -13,6 +13,9 @@
 import org.openas2.message.Message;
 
 public interface ICryptoHelper {
+    static final int JCE_LIMITED_MAX_LENGTH = 128;
+    static final String JCE_LIMITATION_ERROR = "OpenAS2 needs `Java Cryptography Extension` (JCE) Unlimited Strength Jurisdiction Policy Files!\nPlease verify it is installed in your Java installation. You can download it from Oracle Web site.";

```

Comment:
Why the backquotes?
Would be better like this:
"OpenAS2 needs Java Cryptography Extension (JCE) Unlimited Strength Jurisdiction Policy Files!\nPlease verify it is installed in your Java installation. You can download it from Oracle Web site."

Reply:
I would:
- replace Oracle with "official web site", since trade marks and other legal things.
- inline JCE_LIMITATION_ERROR, since as a constant it's useless.